nhost/compose/config.go

```diff
@@ -229,32 +207,19 @@ func (c Config) DashboardURL() string {
 }
 
 func (c Config) mailhogServiceEnvs() env {
-	authEnv := c.authServiceEnvs()
+	smtpConf := c.conf.Provider().GetSmtp()
 
-	e := env{
-		"SMTP_HOST":   authEnv[envAuthSmtpHost],
-		"SMTP_PORT":   authEnv[envAuthSmtpPort],
-		"SMTP_PASS":   authEnv[envAuthSmtpPass],
-		"SMTP_USER":   authEnv[envAuthSmtpUser],
-		"SMTP_SECURE": authEnv[envAuthSmtpSecure],
-		"SMTP_SENDER": authEnv[envAuthSmtpSender],
-	}
-
-	e.merge(c.serviceConfigEnvs(SvcMailhog))
-	e.mergeWithSlice(c.dotenv)
-	return e
-}
-
-func (c Config) runMailhogService() bool {
-	if conf, ok := c.nhostConfig.Services[SvcMailhog]; ok && conf != nil {
-		if conf.NoContainer {
-			return false
-		}
-	}
-
-	authEnv := c.authServiceEnvs()
-
-	return authEnv[envAuthSmtpHost] == SvcMailhog
+	return env{
+		"SMTP_HOST":   smtpConf.GetHost(),
+		"SMTP_PORT":   fmt.Sprint(smtpConf.GetPort()),
+		"SMTP_PASS":   smtpConf.GetPassword(),
+		"SMTP_USER":   smtpConf.GetUser(),
+		"SMTP_SECURE": fmt.Sprint(smtpConf.GetSecure()),
+		"SMTP_SENDER": smtpConf.GetSender(),
+	}.merge(
+		// global environment variables from config
+		c.globalEnvs(),
+	)
```

Comment:
Can you elaborate?

Reply:
They may override envs that are set from the configuration for specific service. Like in the example above, first we set `SMTP_HOST` with value from `Provider.Smtp.Host`. It can be overriden in global envs. Question is, where shall I put global envs, as first (that can be overriden) or last (override others) element in the chain?

Reply:
> It can be overriden in global envs

That shouldn't be supported. It won't work in the cloud so it shouldn't work in the cli either.

Reply:
How global envs are supposed to be used in CLI?

Reply:
I don't know what you mean but how env vars are supposed to work was described in this comment:

https://github.com/nhost/cli/pull/542#discussion_r1083970180